ui/apps/platform/src/Containers/AppPage.tsx

```diff
@@ -20,14 +20,21 @@ function AppPage(): ReactElement {
             <AppPageTitle />
             <AppPageFavicon />
             <Switch>
-                <Route path={loginPath} component={LoginPage} />
-                <Route
-                    path={authorizeRoxctlPath}
-                    render={(props) => <LoginPage {...props} authorizeRoxctlMode />}
-                />
-                <Route path={testLoginResultsPath} component={TestLoginResultsPage} />
-                <Route path={authResponsePrefix} component={LoadingSection} />
-                <Route component={AuthenticatedRoutes} />
+                <Route path={loginPath}>
+                    <LoginPage authProviders={undefined} />
+                </Route>
+                <Route path={authorizeRoxctlPath}>
+                    <LoginPage authorizeRoxctlMode authProviders={undefined} />
+                </Route>
```

Comment:
Not a change request, but what are your thoughts on the following alternatives?

- Removing `isRequired` from the `authProviders` prop in the prop types
- Passing `[]` as the initial value instead of `undefined`. (This might be a change in runtime behavior, but conforms with the expected type that is used within the `LoginPage` component)

Reply:
I like option 2 based off what I wrote here https://github.com/stackrox/stackrox/pull/12183#discussion_r1765539168